subsquid/src/processors/pablo.ts

```diff
@@ -28,25 +29,41 @@ import { divideBigInts, encodeAccount } from "../utils";
 import {
   getLatestPoolByPoolId,
   getOrCreateHistoricalAssetPrice,
+  getOrCreateFeeApr,
   getOrCreatePabloAsset,
+  getOrCreatePabloLpToken,
   saveAccountAndEvent,
   saveActivity,
   saveEvent
 } from "../dbHelper";
+import { chain } from "../config";
 
 interface PoolCreatedEvent {
   owner: Uint8Array;
   poolId: bigint;
   assetWeights: [bigint, number][];
+  lpTokenId: string;
 }
 
-function getPoolCreatedEvent(event: PabloPoolCreatedEvent): PoolCreatedEvent {
+async function getPoolCreatedEvent(event: PabloPoolCreatedEvent): Promise<PoolCreatedEvent> {
   const { owner, poolId, assetWeights } = event.asV10005;
-  return {
+
+  const wsProvider = new WsProvider(chain());
+  const api = await ApiPromise.create({ provider: wsProvider });
+
+  // TODO: get lpTokenId from the event
+  // This is a temporary solution, and does not affect performance as there are only 3 pools created
+  const pool = (await api.query.pablo.pools(poolId))?.toJSON();
```

Comment:
Why not just take from the event? or hard code for existing pools?

Reply:
yes, this will be taken from the event, which I added recently (https://github.com/ComposableFi/composable/pull/3168) but it's not in the runtime yet. Hardcoding can be a better temporary option though

Reply:
Fixed, need re-approval @vimukthi-git

Reply:
Don't know if there are events available for existing pool that were created using the migration though